package.json

```diff
@@ -33,9 +33,11 @@
     "eslint-plugin-filenames": "^1.3.2",
     "eslint-plugin-i18n-text": "^1.0.1",
     "eslint-plugin-import": "^2.25.2",
+    "eslint-plugin-jsx-a11y": "^6.6.0",
     "eslint-plugin-no-only-tests": "^2.6.0",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-rule-documentation": ">=1.0.0",
+    "jsx-ast-utils": "^3.3.2",
```

Comment:
This provides utility methods that make parsing JSX elements much more pleasant. I found it through `jsx-a11y-plugin`. 

[Repo for jsx-ast-utils](https://github.com/jsx-eslint/jsx-ast-utils)

Reply:
do u also need to use the typescript ast?

Reply:
It seems yes, as long as the `.eslintrc.json` wherever the linters get run is configured with the typescript parser:

```
 "parser": "@typescript-eslint/parser",
```

I tested this branch out in dotcom by running:

```
npm i git://github.com/github/eslint-plugin-github.git#kh-add-a11y --save
```

(^today I learned how to install a package from a GitHub branch!!!)


I see the lint error show up as expected in my editor!

<img width="1161" alt="Screenshot of my VSCode editor in dotcom which flags an eslint violation from this rule" src="https://user-images.githubusercontent.com/16447748/179822266-bfe911e7-1651-43db-aeb0-d302c75faeee.png">

Reply:
🕺 ! YAY thanks for checking